lib/bookmarks.coffee

```diff
@@ -11,7 +11,7 @@ class ReactBookmarks
       'bookmarks:jump-to-previous-bookmark': @jumpToPreviousBookmark
       'bookmarks:clear-bookmarks': @clearBookmarks
 
-    @markerLayer = @editor.addMarkerLayer({maintainHistory: true})
+    @markerLayer = @editor.addMarkerLayer()
```

Comment:
Totally. 👍

I'd like this change to be completely compatible with master, though, so I am not going to add this feature just yet and keep bookmarks not persistent for the time being (as they were before this PR). We can revisit this later by implementing a deserializer that calls `getMarkerLayer` instead of creating a new one.